### PR TITLE
Feat  add monitoring to chart

### PR DESCRIPTION
**Pull request by osamamagdy**
Add Prometheus and grafana to the chart

What kind of changes does this PR include?

-   [X] A new feature
-   [X] Additional documentation

Checklist:

-   [X] All the contributions made are solely the work of me and my co-authors
-   [X] I tested the changes in this PR (if applicable)
-   [X] I added tests to ensure my change works (if applicable)
-   [X] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: helm/wrongsecrets-ctf-party/values.yaml [hunk 1]
```diff
@@ -195,7 +178,7 @@ wrongsecrets:
   # -- Optional environment variables to set for each Wrongsecrets instance (see: https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
   env:
     - name: K8S_ENV
-      value: k8s
+      value: gcp
```
Comment (bendehaan) [inline helm/wrongsecrets-ctf-party/values.yaml:181]: Same here

File: helm/wrongsecrets-ctf-party/values.yaml [hunk 2]
```diff
@@ -94,7 +94,7 @@ balancer:
     REACT_APP_S3_BUCKET_URL: "s3://funstuff" #the s3 bucket you use for the aws challenges, don't forget to make it accessible!
     REACT_APP_GCP_BUCKET_URL: "gs://funstuff" #the gcp bucket you use for the gcp challenges, don't forget to make it accessible!
     REACT_APP_AZ_BLOB_URL: "az://funstuff" #the azure blob storage you use for the azure challenges, don't forget to make it accessible!
-    K8S_ENV: "k8s" #or 'aws', 'azure', or 'gcp'
+    K8S_ENV: "gcp" #or 'aws', 'azure', or 'gcp'
```
Comment (bendehaan) [inline helm/wrongsecrets-ctf-party/values.yaml:97]: Is this change intended?

File: helm/wrongsecrets-ctf-party/values.yaml [hunk 3]
```diff
@@ -170,7 +153,7 @@ wrongsecrets:
   # -- Specify a custom Wrongsecrets config.yaml. See the Wrongsecrets Docs for any needed ENVs: https://github.com/OWASP/wrongsecrets
   # @default -- See values.yaml for full details
   config: |
-    K8S_ENV: k8s
+    K8S_ENV: gcp
```
Comment (bendehaan) [inline helm/wrongsecrets-ctf-party/values.yaml:156]: Same here

File: helm/wrongsecrets-ctf-party/values.yaml (hunk 2 above)
Comment (osamamagdy) [reply helm/wrongsecrets-ctf-party/values.yaml:97]: No it was not sorry for that

Comment (osamamagdy) [reply helm/wrongsecrets-ctf-party/values.yaml]: Edited

File: helm/wrongsecrets-ctf-party/values.yaml (hunk 1 above)
Comment (osamamagdy) [reply helm/wrongsecrets-ctf-party/values.yaml]: Edited

File: helm/wrongsecrets-ctf-party/values.yaml (hunk 3 above)
Comment (osamamagdy) [reply helm/wrongsecrets-ctf-party/values.yaml]: Edited

File: wrongsecrets-balancer/src/app.js [hunk 4]
```diff
@@ -2,51 +2,11 @@ const express = require('express');
...
-const promClient = require('prom-client');
-const basicAuth = require('basic-auth-connect');
-const onFinished = require('on-finished');
-
 const { get, extractTeamName } = require('./config');
 const { logger } = require('./logger');
 
 const app = express();
 
-if (get('metrics.enabled')) {
```
Comment (commjoen) [inline wrongsecrets-balancer/src/app.js:14]: why delete the metrics endpoint and config from the balancer?

**Review by bendehaan — APPROVED** [review]
